### PR TITLE
Docker test tagged as DockerTest.

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

**Comment by cjllanwarne** [comment]
:+1:

File: src/test/scala/cromwell/RetryableCallsSpec.scala [hunk 1]
```diff
@@ -31,7 +31,8 @@ class RetryableCallsSpec extends CromwellTestkitSpec with Mockito {
 
       val workflowId = waitForHandledMessagePattern(pattern = "transitioning from Running to Succeeded") {
         EventFilter.info(pattern = s"persisting status of do_scatter:0:2 to Starting", occurrences = 1).intercept {
-          messageAndWait[WorkflowManagerSubmitSuccess](SubmitWorkflow(SampleWdl.PrepareScatterGatherWdl.asWorkflowSources())).id
+          messageAndWait[WorkflowManagerSubmitSuccess](SubmitWorkflow(
+            SampleWdl.PrepareScatterGatherWdl.asWorkflowSources(workflowOptions = """{"read_from_cache":false}"""))).id
```
Comment (mcovarr) [inline src/test/scala/cromwell/RetryableCallsSpec.scala]: hmm this should maybe be the default

Comment (cjllanwarne) [inline src/test/scala/cromwell/RetryableCallsSpec.scala]: You missed the tech talk!

**Comment by kshakir** [comment]
#466 Fixed one of the tests. Updated this PR to just tag the docker test.

**Comment by kshakir** [comment]
@Horneth for review.

**Comment by Horneth** [comment]
:+1:

**Pull request merged**